e2e/tests/router/navigate-by-link.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { expect, test } from '@playwright/test'
+import { BASE } from '../../utils/env'
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('router/navigate-by-link.html')
+})
+
+test('TODO', async ({ page }) => {
+  // TODO
+  await expect(page).toHaveURL(`${BASE}router/navigate-by-link.html`)
```

Comment:
Made some updates. It would be helpful if you can help to complete the test cases. If you don't have time, I'll add it tmr.

Reply:
Done